element_deeplabcut/dlc.py

```diff
@@ -476,23 +568,26 @@ class BodyPartPosition(dj.Part):
         """
 
     def make(self, key):
+        """.populate() method will launch training for each PoseEstimationTask"""
         from .readers import dlc_reader
 
         # ID model and directories
         dlc_model = (Model & key).fetch1()
-        task_mode, output_dir = (PoseEstimationTask & key).fetch1('task_mode', 'pose_estimation_output_dir')
-
-        output_dir = find_full_path(get_dlc_root_data_dir(), output_dir)
-
+        task_mode, params, output_dir = (PoseEstimationTask & key
+                                         ).fetch1('task_mode', 'pose_estimation_params',
+                                                  'pose_estimation_output_dir')
+        if not Path(output_dir).exists():
+            output_dir = find_full_path(get_dlc_root_data_dir(), output_dir)

```

Comment:
```suggestion
        output_dir = find_full_path(get_dlc_root_data_dir(), output_dir)
```
There is no need for that `if`